kvbc/include/v4blockchain/detail/column_families.h

```diff
@@ -17,11 +17,6 @@
 
 namespace concord::kvbc::v4blockchain::detail {
 
-// Blockchain
-inline const auto BLOCKS_CF = std::string{"v4_blocks"};
-inline const auto ST_CHAIN_CF = std::string{"v4_st_chain"};
-inline const auto LATEST_KEYS_CF = std::string{"v4_latest_keys"};
-inline const auto IMMUTABLE_KEYS_CF = std::string{"v4_immutable_keys"};
-inline const auto CATEGORIES_CF = std::string{"v4_categories"};
+#include "storage/db_column_families.h"
```

Comment:
As the timestamp changes are not used so the key comparator setting may not be needed. So it will be good to keep the column families here.

Reply:
yes, we have debt with the compaction filter that we need to set, so need to evaluate once we've implemented it.